.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: Python application
```

Comment:
You'll just need to update the tests badge in the read me because I use the python-checks.yml file.

Reply:
Good catch, updated this